### PR TITLE
Add in QAE module for environment variable detection 

### PR DESCRIPTION
**Pull request by matthewford**

### CODE REVIEW

File: config/environments/staging.rb [hunk 1]
```diff
@@ -1,95 +0,0 @@
-Rails.application.configure do
```
Comment (rusllonrails) [inline config/environments/staging.rb]: @matthewford why we are going to remove config/environments/staging.rb ?

**Comment by rusllonrails** [comment]
Looks good :+1: 
One small question above ^

File: config/environments/staging.rb (hunk 1 above)
Comment (matthewford) [inline config/environments/staging.rb]: @rusllonrails as per 12factor, all env run in production with ENV variables for the different settings.

Comment (rusllonrails) [inline config/environments/staging.rb]: Ok. fine

**Pull request merged**